pkg/ddc/alluxio/transform.go

```diff
@@ -275,6 +276,9 @@ func (e *AlluxioEngine) transformMasters(runtime *datav1alpha1.AlluxioRuntime,
 
 	value.Master.HostNetwork = true
 
+	nodeSelector := e.transformMasterSelector(runtime)
+	value.Master.NodeSelector = nodeSelector
```

Comment:
I suggest to check if the nodeSelector is empty.

Reply:
thx, fixed.